src/sage/graphs/generic_graph.py

```diff
@@ -1926,7 +1936,10 @@ def adjacency_matrix(self, sparse=None, vertices=None, *, base_ring=None, **kwds
                 sparse = False
 
         if vertices is None:
-            vertices = self.vertices(sort=True)
+            try:
+                vertices = self.vertices(sort=True)
+            except TypeError:
+                raise TypeError ("Vertex labels are not comparable. You must specify an ordering using parameter ``vertices``")
```

Comment:
Please use the 80 columns mode here like:
```diff
-                raise TypeError ("Vertex labels are not comparable. You must specify an ordering using parameter ``vertices``")
+                raise TypeError("Vertex labels are not comparable. You must "
                                 "specify an ordering using parameter "
                                 "``vertices``")
```

Reply:
done in a7f0659